client/components/select-dropdown/test/index.js

```diff
@@ -0,0 +1,136 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+import { assert } from 'chai';
+import { shallow, mount } from 'enzyme';
+import sinon from 'sinon';
```

Comment:
We also have the option of using a sinon sandbox so we can avoid having to manually restore stubs:
```
import { useSandbox } from 'test/helpers/use-sinon';
//...
let sandbox;
useSandbox( newSandbox => {
	sandbox = newSandbox;
});
//...
const stub = sandbox.stub( React.Component.prototype, 'setState' );
```